docs/locations.md

```diff
@@ -0,0 +1,165 @@
+- Have some Ark backups go to a bucket in an eastern USA region, and others go to a bucket in a western USA region
+- For volume providers that support it (e.g. Portworx), have some snapshots be stored locally on the cluster and have others be stored in the cloud
+
+Additionally, as we looked ahead to a major feature on our roadmap, backup replication, we knew that we'd need Ark to be able to support multiple possible storage locations.
+
+## Overview
+
+In Ark v0.10 we got rid of the `Config` custom resource, and replaced it with two new custom resources, `BackupStorageLocation` and `VolumeSnapshotLocation`. The new resources directly replace the legacy `backupStorageProvider` and `persistentVolumeProvider` sections of the `Config` resource, respectively. 
+
+Now, the user can pre-define more than one possible `BackupStorageLocation` and more than one `VolumeSnapshotLocation`, and can select *at backup creation time* the location in which the backup and associated snapshots should be stored. 
+
+A `BackupStorageLocation` is defined as a bucket, a prefix within that bucket under which all Ark data should be stored, and a set of additional provider-specific fields (e.g. AWS region, Azure storage account, etc.) The [API documentation][1] captures the configurable parameters for each in-tree provider.
+
+A `VolumeSnapshotLocation` is defined entirely by provider-specific fields (e.g. AWS region, Azure resource group, Portworx snapshot type, etc.) The [API documentation][2] captures the configurable parameters for each in-tree provider.
+
+Additionally, since multiple `VolumeSnapshotLocations` can be created, the user can now configure locations for more than one volume provider, and if the cluster has volumes from multiple providers (e.g. AWS EBS and Portworx), all of them can be snapshotted in a single Ark backup.
+
+## Limitations / Caveats
+
+- For volume snapshots, you are still limited by where your provider allows you to create snapshots. For example, AWS and Azure do not allow you to create a volume snapshot in a different region than where the volume is. If you try to take an Ark backup using a volume snapshot location with a different region than where your cluster's volumes are, the backup will fail.
+
+- Each Ark backup has one `BackupStorageLocation`, and one `VolumeSnapshotLocation` per volume provider. It is not possible (yet) to send a single Ark backup to multiple backup storage locations simultaneously, and similarly for volume snapshots. However, you can always set up multiple scheduled backups that differ only in the storage locations used if redundancy of backups across locations is important.
+
+- Cross-provider snapshots are not supported. If you have a cluster with more than one type of volume (e.g. EBS and Portworx), but you only have a `VolumeSnapshotLocation` configured for EBS, then Ark will **only** snapshot the EBS volumes.
```

Comment:
Trying to clear up a bit to my current understanding (which may be off) which is that you can have a backup span multiple VSLs, they are just constrained by the volume's underlying provider.

```suggestion
- Cross-provider snapshots are not supported. If you have a cluster with more than one type of volume then Ark will **only** snapshot the volumes for their underlying provider. As an example, if we had a cluster with Portworx and EBS volumes, and a `VolumeStorageLocation` defined for each provider, the EBS volumes would only be snapshot by the EBS `VolumeStorageLocation` and the Portworx would only be snapshot by the Portworx `VolumeStorageLocation`.
```